docs/codeql/codeql-cli/creating-and-working-with-codeql-packs.rst

```diff
@@ -40,10 +40,12 @@ The ``codeql pack init`` command creates the directory structure and configurati
 
 Modifying an existing QL pack to create a CodeQL pack
 -----------------------------------------------------
-If you already have a ``qlpack.yml`` file, you can edit it manually to be a CodeQL pack.
+If you already have a ``qlpack.yml`` file, you can edit it manually to convert it into a CodeQL pack.
 
-#. Edit the name so that it matches the format ``<scope>/<name>``, where ``<scope>`` is the name of the GitHub organization that you will publish to.
-#. In the ``qlpack.yml`` file, include a version property with a semver identifier.
+#. Edit the ``name`` property so that it matches the format ``<scope>/<name>``, where ``<scope>`` is the name of the GitHub organization that you will publish to.
```

Comment:
```suggestion
#. Edit the ``name`` property so that it matches the format ``<scope>/<name>``, where ``<scope>`` is the name of the GitHub organization or user account that you will publish to.
```